tests/integrationv2/Makefile

```diff
@@ -25,12 +25,45 @@ ifeq ($(S2N_CORKED_IO),true)
 	HANDSHAKE_TEST_PARAMS:=--use_corked_io $(HANDSHAKE_TEST_PARAMS)
 endif
 
-.PHONY : all
-all:
+
+define run_tox
 	( \
 	DYLD_LIBRARY_PATH="../../lib/:../testlib/:$(LIBCRYPTO_ROOT)/lib:$$DYLD_LIBRARY_PATH" \
 	LD_LIBRARY_PATH="../../lib/:../testlib/:$(LIBCRYPTO_ROOT)/lib:$$LD_LIBRARY_PATH" \
 	S2N_INTEG_TEST=1 \
 	PATH="../../bin":$(LIBCRYPTO_ROOT)/bin:$(PATH) \
+	TOX_TEST_NAME=$(1) \
 	tox \
 	)
+endef
+
+test_key_update:
+	$(call run_tox,$@.py)
+test_client_authentication:
+	$(call run_tox,$@.py)
+test_dynamic_record_sizes:
+	$(call run_tox,$@.py)
+test_happy_path:
+	$(call run_tox,$@.py)
+test_session_resumption:
+	$(call run_tox,$@.py)
+test_sni_match:
+	$(call run_tox,$@.py)
+test_well_known_endpoints:
+	$(call run_tox,$@.py)
+test_compatibility_with_oqs_openssl:
+	$(call run_tox,$@.py)
+test_fragmentation:
+	$(call run_tox,$@.py)
+test_hello_retry_requests:
+	$(call run_tox,$@.py)
+test_pq_handshake:
+	$(call run_tox,$@.py)
+test_signature_algorithms:
+	$(call run_tox,$@.py)
+test_version_negotiation:
+	$(call run_tox,$@.py)
+
+.PHONY : test_client_authentication test_dynamic_record_sizes test_key_update test_happy_path est_session_resumption test_sni_match test_well_known_endpoints test_compatibility_with_oqs_openssl test_fragmentation test_hello_retry_requests test_pq_handshake test_signature_algorithms test_version_negotiation
```

Comment:
I think my comment got overwritten by newer commits, but on line 67 test_session_resumption is spelled wrong

Reply:
thanks! fixed.